### PR TITLE
Remove PHP 5.6 test from CI

### PR DESCRIPTION
**Pull request by yscik**
### Changes proposed in this Pull Request

* Remove the PHP 5.6 test run from Travis. Since Sensei `master` requires PHP 7.0 now, it's failing
* Minimum PHP version is not bumped for now since it could cause issues with auto-updating from WC.com

### Testing instructions

* Travis green

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ vendor
 build
 assets/dist
 /*.zip
+.idea
```
Comment (renatho) [inline .gitignore:7]: Maybe should it be part of another PR?

**Review by donnapep — APPROVED** [review]

**Review by renatho — APPROVED** [review]
Looks good to me!
I just saw a new ignored folder that I think you added accidentally, but it won't hurt if you want to introduce it 🙂. I think it's from the IDE, right?

File: .gitignore (hunk 1 above)
Comment (yscik) [reply .gitignore:7]: Yeah I sneaked that in because IDE files were showing up. It's already in the main Sensei repo - we can sync this file too at some point.

Comment (jom) [reply .gitignore:7]: Ahh, I probably accidentally removed it as I copied and pasted this file across all repos. 

I have `.idea` in my global gitignores file.
